cirq-google/cirq_google/engine/engine_processor.py

```diff
@@ -105,19 +111,23 @@ def get_sampler(
         processor = self._inner_processor()
         # If a run_name or config_alias is not provided, initialize them
         # to the Processor's default values.
```

Comment:
update the comment here. "If a run_name or config_alias"  isn't neither of them provided?